docs/griptape-framework/drivers/prompt-drivers.md

```diff
@@ -209,7 +209,7 @@ agent = Agent(
     config=StructureConfig(
         global_drivers=StructureGlobalDriversConfig(
             prompt_driver=AnthropicPromptDriver(
-                model="claude-2",
+                model="claude-3-opus-20240229",
```

Comment:
I guess I can't add comments to unchanged code. :(

But above the docs need to change to reference the Anthropic `messages` instead of `completions`.

Also: can we add an example in here on how to initialize the driver with the [Structure PR](https://github.com/griptape-ai/griptape/pull/702/files#diff-388135d66c742b44c2b3cd526877c0d0d34ca1938daa41523ade65a86a3f24f5R21)? I don't fully understand the functional change and how it would impact how we use the Anthropic drivers?

Reply:
Great catch on `messages`.

Reply:
Can you expand a bit more on what you're looking for?

Reply:
if I wanted to specify my own Anthropic client and pass it to my driver, how would I do that now? I saw that it changed in that PR and wondering if there's anything functional we need to provide examples for users?

Reply:
TBH there aren't many uses I can think of for passing in your own client, but we keep interface open to do so. The change you're referring to in the other PR isn't a change in functionality, only a change to the type hint.

Reply:
Got it, makes sense. No other changes needed then.